.github/workflows/format-lint-test.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Use Node.js
       uses: actions/setup-node@v4
       with:
-        node-version: '20.x'
+        node-version: '21.x'
```

Comment:
Any specific reason for this change? `20.x` is the current LTS, no?

Reply:
I had problems with format not working locally so I changed this one, (eventually it was tsconfig.json that was formatted incorrectly so I added it to the prettierignore). I can revert this change I thought it was fine because it still worked and was newer

Reply:
changed it back to 20.x